### ISSUE TITLE
NVIM Marksman cmp lists and works with local wikistyle heading [[#anchor]] anchors, but not with path style anchor refrenchong [title](./file.md#anchor)... 

### ISSUE THREAD

**Issue by MikeLemo1** [artempyanykh/marksman#298]
Just like the title says I am able to cmp list and action on local style wiki link `[[#anchor]]` but whe using 
[title](./my_file.md#anchor) it doesn't list nor action on the link

Tested with: ++gd++ ++gf++ ++shift+k++ etc...

as on the nvim installation I've just auto installed and loaded marksmen using `:Mason`

Any idea what went wrong on the way?

**Comment by bcvanmeurs** [comment]
Yeah having the same issue here, `gd` works for local headings, but not for external links.

**Comment by bcvanmeurs** [comment]
I figured out that in my case it doesn't work because I am using a git worktree. When I am in a 'normal' git repo it does just work.

With the worktree I get this in the LSP logs:

> [ERROR][2024-03-27 13:21:02] .../vim/lsp/rpc.lua:734	"rpc"	"/Users/bcvanmeurs/.local/share/nvim/mason/bin/marksman"	"stderr"	'[13:21:02 WRN] <ServerUtil> Workspace folder is bogus: {"root": "RootPath (AbsPath \\"/Users/bcvanmeurs/repos/project-x/main\\")"}\n'

in which `project-x` is the bare git repo and `main` is a worktree.
